integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/IngressSteps.java

```diff
@@ -26,9 +25,31 @@ public IngressSteps(TestContext context) {
         this.context = context;
     }
 
-    @Then("^send cloud events to the ingress at the endpoint with access token:$")
-    public void testIngressEndpoint(String cloudEvent) {
-        String endpoint = BridgeUtils.getOrRetrieveBridgeEndpoint(context);
+    @And("^the Ingress of Bridge \"([^\"]*)\" is available within (\\d+) (?:minute|minutes)$")
+    public void ingressOfBridgeIsAvailableWithinMinutes(String testBridgeName, int timeoutMinutes) {
+        String endpoint = BridgeUtils.getOrRetrieveBridgeEndpoint(context, testBridgeName);
+
+        Awaitility.await().atMost(Duration.ofMinutes(timeoutMinutes))
```

Comment:
question, will similar `atMost` calls cause, that if such timeout is reached, some proper log/exception mentioning timeout reached will appear? Asking because of my experience with `assertj` where `as` needed to be called to get some better error message.

Reply:
there will be a timeout display and the step is already self-explaining (aka `within ... minutes`)

Reply:
perfect